packages/conventional-commit-lint/src/conventional-commit-lint.ts

```diff
@@ -38,6 +46,31 @@ export = (app: Probot) => {
         );
         return;
       }
+
+      // Conventional Commit Lint (unlike most automations) is opt-out, vs.,
+      // opt in. For this reason config is loa
+      let config: Configuration | undefined = undefined;
+      try {
+        config = await getConfigWithDefault<Configuration>(
+          context.octokit,
+          owner,
+          repo,
+          CONFIGURATION_FILE,
+          {},
+          {schema: schema}
```

Comment:
I'm of the opinion that we should not throw an error here if the schema doesn't match. Let the presubmit check that you're going to add surface the error. Alternatively, use `@google-automations/issue-utils` to open an issue on the repository that the config is invalid (otherwise, to the repo owner, you don't know there's an error).

Example: if the user erroneously adds a config file with extra entries, then this whole event handler does not run.

Reply:
> I'm of the opinion that we should not throw an error here if the schema doesn't match. Let the presubmit check that you're going to add surface the error.

I was intending to let the failed check surface config format errors, the `try`/`catch` is to catch other types of errors (missing permissions in bot).

Reply:
I still think `log.error` is not right because as a bot, it is the correct behavior.

Reply:
> I still think log.error is not right because as a bot, it is the correct behavior.

I agree, I've switched to `logger.warn`.

Reply:
Update: Ah sorry I somehow think the exception is thrown when the schema is wrong.

I re-read bot-config-utils and realized the exception is thrown only when something bad happens.

Reply:
Ok, `warn` also works for me